fsociety/__main__.py

```diff
@@ -112,30 +115,81 @@ def agreement():
             config.set("fsociety", "agreement", "true")
 
 
+subcommands = list()
 for item in MENU_ITEMS:
     items[module_name(item)] = item
-
+    for tool in item.__tools__:
+        subcommands.append(
+            {"parent": item.__name__, "tool": tool, "name": tool.__str__()}
+        )
 commands = list(items.keys()) + list(BUILTIN_FUNCTIONS.keys())
 
 
+def errorhandle(error: Exception, style: Optional[Union[str, Style]] = "red") -> None:
```

Comment:
Let's import this instead of repeating it.

Reply:
not sure that I understand how to import the type for this, assuming that's what you're wanting to be imported, because it's not got a single unified type on it.

Reply:
i don't think it's needed anymore, with the updated Console interface